verticapy/geo.py

```diff
@@ -129,6 +130,59 @@ def create_index(
     )
     return to_tablesample(query, vdf._VERTICAPY_VARIABLES_["cursor"])
 
+# ---#
+def coordinate_converter(vdf: vDataFrame,
+                         x: str, 
+                         y: str,
+                         x0: float = 0.0, 
+                         earth_radius: float = 6371,
+                         reverse: bool = False,):
+    """
+---------------------------------------------------------------------------
+Converts latitude and longitude to Euclidean coordinates x, y or the opposite.
+
+Parameters
+----------
+vdf: vDataFrame
+    input vDataFrame.
+x: str
+    vcolumn used as abscissa (longitude).
+y: str
+    vcolumn used as ordinate (latitude).
+x0: float, optional
+    Initial abscissa.
+earth_radius: float, optional
+    Earth radius.
```

Comment:
It would be very nice if you clarify the unit here; i.e. KM.

Reply:
Done;